tests/reference/runtime-test_list_02-5f7db5f.stderr

```diff
@@ -0,0 +1 @@
+IndexError: list index out of range
```

Comment:
Which index is out of range and what's the range? Useful information to add. I think its possible, we just need to modify the arguments to print call, correct?

Reply:
Yup, that's possible! I will give it a try in this PR, if possible
Otherwise, I will to submit a PR for that after completing this issue: #772.

Reply:
Done!